langchain/src/memory/base.ts

```diff
@@ -21,13 +21,16 @@ export abstract class BaseMemory {
   ): Promise<void>;
 }
 
-export const getInputValue = (inputValues: InputValues) => {
+export const getInputValue = (inputValues: InputValues, inputKey?: string) => {
```

Comment:
can you explain why these changes are needed? If I'm understanding correctly, `inputValues` can only have one key rn, why not just always grab the only value?

Reply:
for conversational agent, there ends up being multiple keys (the memory key and the input key)